lib/hdf5db.py

```diff
@@ -1324,11 +1328,10 @@ def makeNullTermStringAttribute(self, obj, attr_name, strLength, value):
         tid.set_size(strLength)
         tid.set_strpad(h5py.h5t.STR_NULLTERM)
         sid = h5py.h5s.create(h5py.h5s.SCALAR)
-        aid = h5py.h5a.create(obj.id, attr_name, tid, sid)
```

Comment:
Reviewing that change, it appears to have been the result of overzealous linting. Good catch.